src/types/models.ts

```diff
@@ -177,4 +177,11 @@ type Flatten<EventType extends SeamEvent["event_type"]> =
       } & Extract<SeamEvent, { event_type: EventType }>["payload"]
     : never
 
-export type Event = Flatten<SeamEvent["event_type"]>
+export type Event = Flatten<SeamEvent["event_type"]> & {
```

Comment:
This type is a little confusing, the payload is already declared on `SeamEvent`. I assume the resulting type is the right shape if tests pass but maybe there's a more clear way to define it.

The `payload` shouldn't be manually defined but pulled from `SeamEvent`.

Reply:
Oh, interesting, okay. Removing the manually-declared `payload`, the tests fail with a validation error saying that the value returned from the GET request should NOT have `payload` declared (because it's "not defined" within the `SeamEvent` type or whatever)

Reply:
It's defined right there on `CommonDeviceEvent`, but it's not being accounted for in the tests for some reason.

Reply:
I think I see the issue, the response from `/events/get` is a little weird:
```
{
        created_at: '2022-09-28T17:22:36.928Z',
        device_id: 'd8e7003b-7ccf-46c5-a1af-4dead3e656c6',
        event_id: '90d8507b-8860-4593-89f5-6cc76ab280eb',
        event_type: 'device.connected',
        payload: {
          device_id: 'd8e7003b-7ccf-46c5-a1af-4dead3e656c6',
          workspace_id: '46d67e62-9918-43c3-ac33-af194680ae5d',
        },
        workspace_id: '46d67e62-9918-43c3-ac33-af194680ae5d',
      }
```

We should probably filter on Seam Connect and remove the duplicated fields.

@seveibar do we want to remove the hoisted params or the `payload` field?

Reply:
This also applies to Connected Accounts:

```
Result: [
    {
        "account_type":"schlage",
        "connected_account_id":"2b219953-a981-4f96-bbbb-9d181350319b",
        "created_at":"2022-09-28T17:58:44.321Z",
        "errors":[
            
        ],
        "user_identifier":{
            "email":"954bbfc9-8285-4cb0-bb0e-88a52d618060@example.com"
        }
    },
    {
        "account_type":"august",
        "connected_account_id":"946ab456-ec0a-46aa-bce6-b03d8bcaadba",
        "created_at":"2022-09-28T17:58:44.377Z",
        "errors":[
            
        ],
        "user_identifier":{
            "email":"user-3@example.com"
        }
    }
]

Validation errors: [
    {
        "instancePath":"/0",
        "keyword":"additionalProperties",
        "message":"must NOT have additional properties",
        "params":{
            "additionalProperty":"errors"
        },
        "schemaPath":"#/additionalProperties"
    }
]
```

Reply:
Hey @seveibar, what do you think here?

Reply:
woops sorry for late response,

yes we should remove `payload` from `/events/get`!